pro/src/new_components/VenueForm/Address/__specs__/Address.spec.tsx

```diff
@@ -0,0 +1,158 @@
+import '@testing-library/jest-dom'
+
+import { render, screen, waitFor } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { Formik } from 'formik'
+import React from 'react'
+import * as yup from 'yup'
+
+import { apiAdresse } from 'apiClient/adresse'
+import { IVenueFormValues } from 'new_components/VenueForm'
+import { SubmitButton } from 'ui-kit'
+
+import { Address, validationSchema as addressValidationSchema } from '../'
+
+const mockAdressData = [
+  {
+    address: '12 rue des lilas',
+    city: 'Lyon',
+    id: '1',
+    latitude: 11.1,
+    longitude: -11.1,
+    label: '12 rue des lilas 69002 Lyon',
+    postalCode: '69002',
+  },
+  {
+    address: '12 rue des tournesols',
+    city: 'Paris',
+    id: '2',
+    latitude: 22.2,
+    longitude: -2.22,
+    label: '12 rue des tournesols 75003 Paris',
+    postalCode: '75003',
+  },
+]
+
+jest.mock('apiClient/adresse', () => {
+  return {
+    ...jest.requireActual('apiClient/adresse'),
+    default: {
+      getDataFromAddress: jest.fn(),
+    },
+  }
+})
+
+const renderAddress = async ({
+  initialValues,
+  onSubmit = jest.fn(),
+}: {
+  initialValues: Partial<IVenueFormValues>
+  onSubmit: () => void
+}) => {
+  const validationSchema = yup.object().shape(addressValidationSchema)
+  const rtlReturns = await render(
+    <Formik
+      initialValues={initialValues}
+      onSubmit={onSubmit}
+      validationSchema={validationSchema}
+    >
+      {({ handleSubmit }) => (
+        <form onSubmit={handleSubmit}>
```

Comment:
ici tu peux juste utiliser le <Form> de formik ça marche pareil
```suggestion
        <Form>
```